src/DsaGlobal.cc

```diff
@@ -317,43 +314,41 @@ bool ContextSensitiveGlobalAnalysis::runOnModule(Module &M) {
   CalleeCallerMapping callee_caller_map;
   for (auto it = scc_begin(&m_cg); !it.isAtEnd(); ++it) {
     auto &scc = *it;
-    for (CallGraphNode *cgn : scc) {
+
+    std::vector<CallGraphNode *> cgns = call_graph_utils::SortedCGNs(scc);    
```

Comment:
Again, why not normalize the order of nodes in the call graph after construction so that everything is already ordered before the client starts using the call graph